utils/parseEditorData.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
What is the unused expression you don't want flagged?

Reply:
On the line where I check if the list is ordered or not with a ternary, ESLint wants me to assign that to a variable, but I don't really need to do that cos it seems inefficient and could be a waste of space